### DISCUSSION TITLE
XDM extension

### DISCUSSION THREAD

**Discussion by maaz2456** [subhra74/xdm#1023] [Q&A]
![Capture](https://user-images.githubusercontent.com/128172461/230057165-3e4a5d63-24e1-4f7d-819b-e5bd40a85b2a.PNG)
XDM extension is not installing on chrome.

**Comment by rinelfi-git** [comment]
seems like XDM manifest version is no longer supported by new browser, so they removed it from their repos

**Comment by ghost** [comment]
![Screenshot from 2023-09-08 20-49-53](https://github.com/subhra74/xdm/assets/144465977/7ba55aa2-be2a-42b5-ae51-5b796c27baf7)
Seems like chrome web store has deleted the extension on 2022-09-21 thats why you are getting a error.
You can still download the extension from here: https://chrome-stats.com/d/dkckaoghoiffdbomfbbodbbgmhjblecj

**Reply by CodexUnico** [reply]
Could you also upload it somewhere else, where we wont have to sign in?? how about here on github??

**Comment by amkstone** [comment]


**Reply by ghost** [reply]
@amkstone yes I have given the manual download link above your comment.
